dsmr_backend/migrations/0009_auto_20200103_1230.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.9 on 2020-01-03 11:30
```

Comment:
You might want to rename this file to something more verbose like `0009_emailsettings_email_from.py`.

Don't forget to also rename the record in the Django migrations table, in your local database afterwards.